public_server/public_server.php

```diff
@@ -9,53 +9,46 @@
 use Friendica\Core\Addon;
 use Friendica\Core\Config;
 use Friendica\Core\L10n;
-use Friendica\Model\User;
 use Friendica\Util\DateTimeFormat;
 
 function public_server_install() {
-
 	Addon::registerHook('register_account', 'addon/public_server/public_server.php', 'public_server_register_account');
 	Addon::registerHook('cron', 'addon/public_server/public_server.php', 'public_server_cron');
 	Addon::registerHook('enotify', 'addon/public_server/public_server.php', 'public_server_enotify');
 	Addon::registerHook('logged_in', 'addon/public_server/public_server.php', 'public_server_login');
 }
 
-
 function public_server_uninstall() {
-
 	Addon::unregisterHook('register_account', 'addon/public_server/public_server.php', 'public_server_register_account');
 	Addon::unregisterHook('cron', 'addon/public_server/public_server.php', 'public_server_cron');
 	Addon::unregisterHook('enotify', 'addon/public_server/public_server.php', 'public_server_enotify');
 	Addon::unregisterHook('logged_in', 'addon/public_server/public_server.php', 'public_server_login');
 }
 
-function public_server_register_account($a,$b) {
-
+function public_server_register_account($a, $b) {
 	$uid = $b;
 
-	$days = Config::get('public_server','expiredays');
-	$days_posts = Config::get('public_server','expireposts');
-	if(! $days)
+	$days = Config::get('public_server', 'expiredays');
+	$days_posts = Config::get('public_server', 'expireposts');
+	if (!$days) {
 		return;
+	}
 
-	$r = q("UPDATE user set account_expires_on = '%s', expire = %d where uid = %d",
-		dbesc(DateTimeFormat::utc('now +' . $days . ' days')),
-		intval($days_posts),
-		intval($uid)
-	);
-
-};
-
+	$fields = ['account_expires_on' => DateTimeFormat::utc('now +' . $days . ' days'), 'expire' => $days_posts];
+	dba::update('user', $fields, ['uid' => $uid]);
+}
 
-function public_server_cron($a,$b) {
+function public_server_cron($a, $b) {
 	logger("public_server: cron start");
 
 	require_once('include/enotify.php');
-	$r = q("select * from user where account_expires_on < UTC_TIMESTAMP() + INTERVAL 5 DAY and account_expires_on > '0000-00-00 00:00:00' and
-		expire_notification_sent = '0000-00-00 00:00:00' ");
+	$r = q("select * FROM `user` WHERE `account_expires_on` < UTC_TIMESTAMP() + INTERVAL 5 DAY AND
```

Comment:
Formatting: please set all the SQL keywords in caps or none (but I'd prefer all). Additionally, please wrap all field names in backticks or none (but I'd prefer all).